modules/dnn/test/test_backends.cpp

```diff
@@ -514,7 +514,8 @@ TEST_P(DNNTestNetwork, FastNeuralStyle_eccv16)
 #if defined(HAVE_INF_ENGINE) && INF_ENGINE_VER_MAJOR_GE(2019010000)
     expectNoFallbacksFromIE(net);
 #endif
-    expectNoFallbacksFromCUDA(net);
+    // [TODO] temporarily disabled check for no "fallbacks", since the new engine does not support CUDA yet
+    //expectNoFallbacksFromCUDA(net);
 }
```

Comment:
force old engine here?

Reply:
the model works correctly with the new engine; but the extra check that the model really used all CUDA layers fails. I think, using the new engine to test correct model processing is more important than switching to the old engine.